src/test/crimson/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ target_link_libraries(unittest_seastar_config crimson)
 
 add_executable(unittest_seastar_monc
   test_monc.cc)
-target_link_libraries(unittest_seastar_monc crimson)
+target_link_libraries(unittest_seastar_monc ceph-common crimson)
```

Comment:
> could you instead link it against crimson-common?

It seems to me that the target crimson already links crimson-common, but crimson-common doesn't include "common/pick_address.cc" which defines `get_iface_numa_node(...)`.

After adding it, the compiler now complains about "pick_address.cc:172:6: error: invalid use of incomplete type 'class CephContext'".

Why ceph-common doesn't complain the "incomplete type of CephContext" issue for the same code?

Reply:
we have `#ifdef WITH_SEATAR` in some source source files to disable the code which does not compile or makes sense in seastar world. guess that's why ceph-common has `CephContext` 's definition.

in the case of `pick_address.cc`,  the definition of `CephContext` is used for accessing `_conf`. so i think we should `#include "common/ceph_context.h"` explicitly instead of relying on other included headers to do so. we can add a separated commit to address this.

and make sure it works as expected even if `WITH_SEASTAR` is defined.